src/Entities/Payments/SofortPayment.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Entities\Payments;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @since 6.0
+ *
+ * @ORM\Table(name="payment_sofort")
```

Comment:
I vaguely remember agreeing on each domain (donation, membership) having its own payment data table, so this should probably be `donation_payment_sofort`. Even when memberships are never paid with sofort, at least our database table names are consistent.